app/views/ctc/questions/claim_eitc/edit.html.erb

```diff
@@ -0,0 +1,7 @@
+<% content_for :form_question, t("views.ctc.questions.claim_eitc.title") %>
+
+<% content_for :form_help_text do %>
+  <p><%= t("views.ctc.questions.claim_eitc.p1") %></p>
```

Comment:
i prefer to see multiple paragraphs of text in the yml as an array, but i dont know if we have specific guidance over whether it must be one way or the other to merge.